template linting error in quickguide example

going through the guides today, and noticed that this snippet

```
<h2>{{@title}}</h2>

<ul>
  {{#each @people as |person|}}
    <li {{action "showPerson" person}}>{{person}}</li>
  {{/each}}
</ul>
```

source : https://guides.emberjs.com/release/getting-started/quick-start/#toc_click-events

is causing a template linting error:

```
$ npm run lint:hbs

> ember-quickstart@0.0.0 lint:hbs /home/stefan/dev/projects/ember-quickstart
> ember-template-lint .

/home/stefan/dev/projects/ember-quickstart/app/templates/components/people-list.hbs
  5:8  error  Interaction added to non-interactive element  no-invalid-interactive

✖ 1 problems (1 errors, 0 warnings)

```

--- a/guides/release/getting-started/quick-start.md
+++ b/guides/release/getting-started/quick-start.md
@@ -222,7 +222,7 @@ So far, your application is listing data,
 but there is no way for the user to interact with the information.
 In web applications you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `PeopleList` component.
+First, add a `<button>` with an `action` helper to the `li` in your `PeopleList` component.
 
 ```handlebars {data-filename="app/templates/components/people-list.hbs"}
 <h2>{{@title}}</h2>

--- a/guides/release/getting-started/quick-start.md
+++ b/guides/release/getting-started/quick-start.md
@@ -229,7 +229,7 @@ First add an `action` helper to the `li` in your `PeopleList` component.
 
 <ul>
   {{#each @people as |person|}}
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/release/getting-started/quick-start.md
+++ b/guides/release/getting-started/quick-start.md
@@ -229,7 +229,7 @@ First add an `action` helper to the `li` in your `PeopleList` component.
 
 <ul>
   {{#each @people as |person|}}
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v2.13.0/getting-started/quick-start.md
+++ b/guides/v2.13.0/getting-started/quick-start.md
@@ -249,7 +249,7 @@ So far, your application is listing data, but there is no way for the user
 to interact with the information.
 In web applications, you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First , add a `<button>` with an `action` helper to the `li` in your `people-list` component.
 
 ```handlebars {data-filename=app/templates/components/people-list.hbs data-diff="-5,+6"}
 <h2>{{title}}</h2>

--- a/guides/v2.13.0/getting-started/quick-start.md
+++ b/guides/v2.13.0/getting-started/quick-start.md
@@ -257,7 +257,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/v2.13.0/getting-started/quick-start.md
+++ b/guides/v2.13.0/getting-started/quick-start.md
@@ -257,7 +257,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v2.14.0/getting-started/quick-start.md
+++ b/guides/v2.14.0/getting-started/quick-start.md
@@ -249,7 +249,7 @@ So far, your application is listing data, but there is no way for the user
 to interact with the information.
 In web applications, you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First , add a `<button>` with an `action` helper to the `li` in your `people-list` component.
 
 ```handlebars {data-filename=app/templates/components/people-list.hbs data-diff="-5,+6"}
 <h2>{{title}}</h2>

--- a/guides/v2.14.0/getting-started/quick-start.md
+++ b/guides/v2.14.0/getting-started/quick-start.md
@@ -257,7 +257,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/v2.14.0/getting-started/quick-start.md
+++ b/guides/v2.14.0/getting-started/quick-start.md
@@ -257,7 +257,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v2.15.0/getting-started/quick-start.md
+++ b/guides/v2.15.0/getting-started/quick-start.md
@@ -231,7 +231,7 @@ So far, your application is listing data,
 but there is no way for the user to interact with the information.
 In web applications you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First , add a `<button>` with an `action` helper to the `li` in your `people-list` component.
 
 ```handlebars {data-filename=app/templates/components/people-list.hbs data-diff="-5,+6"}
 <h2>{{title}}</h2>

--- a/guides/v2.15.0/getting-started/quick-start.md
+++ b/guides/v2.15.0/getting-started/quick-start.md
@@ -239,7 +239,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v2.15.0/getting-started/quick-start.md
+++ b/guides/v2.15.0/getting-started/quick-start.md
@@ -239,7 +239,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/v2.16.0/getting-started/quick-start.md
+++ b/guides/v2.16.0/getting-started/quick-start.md
@@ -227,7 +227,7 @@ So far, your application is listing data,
 but there is no way for the user to interact with the information.
 In web applications you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First , add a `<button>` with an `action` helper to the `li` in your `people-list` component.
 
 ```handlebars {data-filename=app/templates/components/people-list.hbs data-diff="-5,+6"}
 <h2>{{title}}</h2>

--- a/guides/v2.16.0/getting-started/quick-start.md
+++ b/guides/v2.16.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v2.16.0/getting-started/quick-start.md
+++ b/guides/v2.16.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/v2.17.0/getting-started/quick-start.md
+++ b/guides/v2.17.0/getting-started/quick-start.md
@@ -227,7 +227,7 @@ So far, your application is listing data,
 but there is no way for the user to interact with the information.
 In web applications you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First , add a `<button>` with an `action` helper to the `li` in your `people-list` component.
 
 ```handlebars {data-filename=app/templates/components/people-list.hbs data-diff="-5,+6"}
 <h2>{{title}}</h2>

--- a/guides/v2.17.0/getting-started/quick-start.md
+++ b/guides/v2.17.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v2.17.0/getting-started/quick-start.md
+++ b/guides/v2.17.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/v2.18.0/getting-started/quick-start.md
+++ b/guides/v2.18.0/getting-started/quick-start.md
@@ -227,7 +227,7 @@ So far, your application is listing data,
 but there is no way for the user to interact with the information.
 In web applications you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First , add a `<button>` with an `action` helper to the `li` in your `people-list` component.
 
 ```handlebars {data-filename="app/templates/components/people-list.hbs" data-diff="-5,+6"}
 <h2>{{title}}</h2>

--- a/guides/v2.18.0/getting-started/quick-start.md
+++ b/guides/v2.18.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v2.18.0/getting-started/quick-start.md
+++ b/guides/v2.18.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.0.0/getting-started/quick-start.md
+++ b/guides/v3.0.0/getting-started/quick-start.md
@@ -227,7 +227,7 @@ So far, your application is listing data,
 but there is no way for the user to interact with the information.
 In web applications you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First , add a `<button>` with an `action` helper to the `li` in your `people-list` component.
 
 ```handlebars {data-filename="app/templates/components/people-list.hbs" data-diff="-5,+6"}
 <h2>{{title}}</h2>

--- a/guides/v3.0.0/getting-started/quick-start.md
+++ b/guides/v3.0.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.0.0/getting-started/quick-start.md
+++ b/guides/v3.0.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.1.0/getting-started/quick-start.md
+++ b/guides/v3.1.0/getting-started/quick-start.md
@@ -227,7 +227,7 @@ So far, your application is listing data,
 but there is no way for the user to interact with the information.
 In web applications you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First , add a `<button>` with an `action` helper to the `li` in your `people-list` component.
 
 ```handlebars {data-filename="app/templates/components/people-list.hbs" data-diff="-5,+6"}
 <h2>{{title}}</h2>

--- a/guides/v3.1.0/getting-started/quick-start.md
+++ b/guides/v3.1.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.1.0/getting-started/quick-start.md
+++ b/guides/v3.1.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.10.0/getting-started/quick-start.md
+++ b/guides/v3.10.0/getting-started/quick-start.md
@@ -229,7 +229,7 @@ First add an `action` helper to the `li` in your `PeopleList` component.
 
 <ul>
   {{#each @people as |person|}}
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.10.0/getting-started/quick-start.md
+++ b/guides/v3.10.0/getting-started/quick-start.md
@@ -229,7 +229,7 @@ First add an `action` helper to the `li` in your `PeopleList` component.
 
 <ul>
   {{#each @people as |person|}}
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.2.0/getting-started/quick-start.md
+++ b/guides/v3.2.0/getting-started/quick-start.md
@@ -227,7 +227,7 @@ So far, your application is listing data,
 but there is no way for the user to interact with the information.
 In web applications you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First , add a `<button>` with an `action` helper to the `li` in your `people-list` component.
 
 ```handlebars {data-filename="app/templates/components/people-list.hbs" data-diff="-5,+6"}
 <h2>{{title}}</h2>

--- a/guides/v3.2.0/getting-started/quick-start.md
+++ b/guides/v3.2.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.2.0/getting-started/quick-start.md
+++ b/guides/v3.2.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.3.0/getting-started/quick-start.md
+++ b/guides/v3.3.0/getting-started/quick-start.md
@@ -227,7 +227,7 @@ So far, your application is listing data,
 but there is no way for the user to interact with the information.
 In web applications you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First , add a `<button>` with an `action` helper to the `li` in your `people-list` component.
 
 ```handlebars {data-filename="app/templates/components/people-list.hbs" data-diff="-5,+6"}
 <h2>{{title}}</h2>

--- a/guides/v3.3.0/getting-started/quick-start.md
+++ b/guides/v3.3.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.3.0/getting-started/quick-start.md
+++ b/guides/v3.3.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.4.0/getting-started/quick-start.md
+++ b/guides/v3.4.0/getting-started/quick-start.md
@@ -227,7 +227,7 @@ So far, your application is listing data,
 but there is no way for the user to interact with the information.
 In web applications you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First , add a `<button>` with an `action` helper to the `li` in your `people-list` component.
 
 ```handlebars {data-filename="app/templates/components/people-list.hbs" data-diff="-5,+6"}
 <h2>{{title}}</h2>

--- a/guides/v3.4.0/getting-started/quick-start.md
+++ b/guides/v3.4.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.4.0/getting-started/quick-start.md
+++ b/guides/v3.4.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.5.0/getting-started/quick-start.md
+++ b/guides/v3.5.0/getting-started/quick-start.md
@@ -227,7 +227,7 @@ So far, your application is listing data,
 but there is no way for the user to interact with the information.
 In web applications you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First , add a `<button>` with an `action` helper to the `li` in your `people-list` component.
 
 ```handlebars {data-filename="app/templates/components/people-list.hbs" data-diff="-5,+6"}
 <h2>{{title}}</h2>

--- a/guides/v3.5.0/getting-started/quick-start.md
+++ b/guides/v3.5.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.5.0/getting-started/quick-start.md
+++ b/guides/v3.5.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.6.0/getting-started/quick-start.md
+++ b/guides/v3.6.0/getting-started/quick-start.md
@@ -227,7 +227,7 @@ So far, your application is listing data,
 but there is no way for the user to interact with the information.
 In web applications you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First , add a `<button>` with an `action` helper to the `li` in your `people-list` component.
 
 ```handlebars {data-filename="app/templates/components/people-list.hbs" data-diff="-5,+6"}
 <h2>{{this.title}}</h2>

--- a/guides/v3.6.0/getting-started/quick-start.md
+++ b/guides/v3.6.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each this.people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.6.0/getting-started/quick-start.md
+++ b/guides/v3.6.0/getting-started/quick-start.md
@@ -235,7 +235,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each this.people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.7.0/getting-started/quick-start.md
+++ b/guides/v3.7.0/getting-started/quick-start.md
@@ -226,7 +226,7 @@ So far, your application is listing data,
 but there is no way for the user to interact with the information.
 In web applications you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First , add a `<button>` with an `action` helper to the `li` in your `people-list` component.
 
 ```handlebars {data-filename="app/templates/components/people-list.hbs" data-diff="-5,+6"}
 <h2>{{this.title}}</h2>

--- a/guides/v3.7.0/getting-started/quick-start.md
+++ b/guides/v3.7.0/getting-started/quick-start.md
@@ -234,7 +234,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each this.people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.7.0/getting-started/quick-start.md
+++ b/guides/v3.7.0/getting-started/quick-start.md
@@ -234,7 +234,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each this.people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.8.0/getting-started/quick-start.md
+++ b/guides/v3.8.0/getting-started/quick-start.md
@@ -226,7 +226,7 @@ So far, your application is listing data,
 but there is no way for the user to interact with the information.
 In web applications you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First , add a `<button>` with an `action` helper to the `li` in your `people-list` component.
 
 ```handlebars {data-filename="app/templates/components/people-list.hbs" data-diff="-5,+6"}
 <h2>{{this.title}}</h2>

--- a/guides/v3.8.0/getting-started/quick-start.md
+++ b/guides/v3.8.0/getting-started/quick-start.md
@@ -234,7 +234,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each this.people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.8.0/getting-started/quick-start.md
+++ b/guides/v3.8.0/getting-started/quick-start.md
@@ -234,7 +234,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each this.people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.9.0/getting-started/quick-start.md
+++ b/guides/v3.9.0/getting-started/quick-start.md
@@ -226,7 +226,7 @@ So far, your application is listing data,
 but there is no way for the user to interact with the information.
 In web applications you often want to listen for user events like clicks or hovers.
 Ember makes this easy to do.
-First add an `action` helper to the `li` in your `people-list` component.
+First , add a `<button>` with an `action` helper to the `li` in your `people-list` component.
 
 ```handlebars {data-filename="app/templates/components/people-list.hbs" data-diff="-5,+6"}
 <h2>{{this.title}}</h2>

--- a/guides/v3.9.0/getting-started/quick-start.md
+++ b/guides/v3.9.0/getting-started/quick-start.md
@@ -234,7 +234,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each this.people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}} role="button">{{person}}</li>
+    <li><button {{action "showPerson" person}}>{{person}}</button></li>
   {{/each}}
 </ul>
 ```

--- a/guides/v3.9.0/getting-started/quick-start.md
+++ b/guides/v3.9.0/getting-started/quick-start.md
@@ -234,7 +234,7 @@ First add an `action` helper to the `li` in your `people-list` component.
 <ul>
   {{#each this.people as |person|}}
     <li>{{person}}</li>
-    <li {{action "showPerson" person}}>{{person}}</li>
+    <li {{action "showPerson" person}} role="button">{{person}}</li>
   {{/each}}
 </ul>
 ```

